helix-term/tests/test/write.rs

```diff
@@ -119,8 +119,9 @@ async fn test_write_fail_mod_flag() -> anyhow::Result<()> {
             (
                 Some(":w<ret>"),
                 Some(&|app| {
-                    assert_eq!(&Severity::Error, app.editor.get_status().unwrap().1);
-
+                    if let Some(status) = app.editor.get_status() {
+                        assert_eq!(&Severity::Error, status.1);
+                    }
```

Comment:
Looks like an unrelated change?

Reply:
I made this flaky test fix after [verification build](https://github.com/helix-editor/helix/actions/runs/3459194871) has failed on this test, so from my perspective it is related as without it this PR wouldn't have gone so far.
So I guess it depends on how you define related.